src/app/qgisapp.cpp

```diff
@@ -630,6 +630,7 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, bool skipVersionCh
   }
 
   sInstance = this;
+  Qgis::ogrRuntimeSupport();
```

Comment:
I think this should be moved to main.cpp and a console error output if it returns false (blocking qgis startup)